src/command/DefaultMenus.js

```diff
@@ -253,6 +253,11 @@ define(function (require, exports, module) {
         // editor_cmenu.addMenuItem(Commands.NAVIGATE_JUMPTO_DEFINITION);
         editor_cmenu.addMenuItem(Commands.TOGGLE_QUICK_EDIT);
         editor_cmenu.addMenuItem(Commands.TOGGLE_QUICK_DOCS);
+		editor_cmenu.addMenuDivider();
```

Comment:
please run `npm test` before you push changes, as that will tell you that your indentation here needs fixing.